zest/releaser/release.py

```diff
@@ -98,10 +100,10 @@ def _make_tag(self):
             else:
                 # all commands are needed in order to proceed normally
                 print("Please create a tag for %s yourself and rerun." %
-                      (self.data['version'],))
+                      (version,))
```

Comment:
I would pass `tag` here instead of `version`, to remind the user what the tag format normally is.
I don't think I ever ran into this part of the code though.